app/src/components/market/profile/view.tsx

```diff
@@ -11,17 +12,13 @@ import { Table, TD, TH, THead, TR } from '../../common/table'
 import { SubsectionTitle } from '../../common/subsection_title'
 import { ThreeBoxComments } from '../../common'
 
-interface Props {
+interface Props extends RouteComponentProps<any> {
```

Comment:
I think we can use `{}` instead of `any` here (if I understand correctly, this is the type of `props.match.params`, and we don't have any params we care about)

Reply:
Done